packages/lesswrong/components/users/UsersProfile.tsx

```diff
@@ -44,16 +45,10 @@ const styles = (theme: ThemeType): JssStyles => ({
     paddingRight: 10,
     marginLeft: "auto",
     [theme.breakpoints.down('md')]: {
-      ...(forumTypeSetting.get() === "EAForum" ? {
-        gridTemplateColumns: `${SECTION_WIDTH}px 1fr`,
-        gridTemplateAreas: `
-          'center right'
-        `,
-      } : {}),
+      display: 'block',
```

Comment:
Also decided to keep the middle column centered for all screen widths, and just move the right sidebar info under the bio at mid widths.

Reply:
Good choice